package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@editorjs/code",
-  "version": "2.4.2",
+  "version": "2.5.0",
```

Comment:
why 2.5.0 ? these changes breaks nothing

Reply:
because it is the new feature, not a bugfix